tests/test_app.py

```
@@ -503,6 +503,14 @@ def test_log_telemetry(mocker):
     from app.common.utils import log_telemetry
     from app.core.models import TelemetryLog
 
+    """Test the log_telemetry utility function."""
+    from app.common.utils import log_telemetry
```

Ruff (0.15.1)

[error] Redefinition of unused `log_telemetry` from line 503: `log_telemetry` redefined here

Remove definition: `log_telemetry`

(F811)